app/Http/Controllers/CsvSearchController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Aic\Hub\Foundation\Exceptions\DetailedException;
+use App\Http\Search\Request as SearchRequest;
+use App\Http\Search\CsvResponse as SearchResponse;
+use Illuminate\Support\Facades\Request as RequestFacade;
+use Illuminate\Http\Request;
+use Elasticsearch;
+
+class CsvSearchController extends SearchController
+{
+
+    public function search(Request $request, $resource = null)
+    {
+        $headers = [
+            'Content-Type' => 'text/csv',
+            'Content-Disposition' => 'inline',
+        ];
+        $data = $this->query('getSearchParams', 'getSearchResponse', 'search', $resource);
+        $titles = [];
+        foreach ($data[0] as $key => $value) {
+            $titles[] = $key;
```

Comment:
Since we just want the keys, you can use [array_keys](https://www.php.net/manual/en/function.array-keys), for example:
```php
$titles = array_keys($data[0]);
```